explorer/models/rootNode.ts

```diff
@@ -275,6 +275,8 @@ export class RootNode extends NodeBase {
             registryRootNodes.push(new RegistryRootNode('Azure', "azureRegistryRootNode", this.eventEmitter, this._azureAccount));
         }
 
+        registryRootNodes.push(new RegistryRootNode('Private registries', 'customRootNode', null));
```

Comment:
Feels like this is more of `Attached registries`. In other words, couldn't I use this to attach all types of things - including private/public/custom/Azure? Also - `attached` is consistent with Cosmos DB

Reply:
Right now it only handles certain types of registries, we felt 'private registries' probably indicates that best.  Might change later.